### PR TITLE
Add Advanced Gas Turbine and Limit Original LGT

### PR DESCRIPTION
**Pull request by Steelux8**
- Added an Advanced LGT, made out of HSS-S, that only accepts gas fuels with a fuel value above 800k per bucket;
- Capped the EU/t output of the regular LGT at 8192 EU/t, regardless of dynamo (it will never explode);

I have made this implementation as a countermeasure for Benzene, when used in IV. Due to some multiblocks, such as the TGS and the ICO, this MV powergen can be stretched to IV without a lot of setup, and it turns out to be really powerful.

In summary, the LGT, which is HV-tier, will be capped to the EU/t of a single IV amp. It will not explode if the turbine inside is too powerful, it will just void the excess energy. To produce more than this from one turbine, the Advanced LGT is implemented here, but it's made out of HSS-S (around 20 Osmium and 40 Iridium per turbine). Additionally, the very strong mar-ce turbine material can also have a countermeasure here, since it's too powerful for the original LGT.

### CODE REVIEW

File: src/main/java/gregtech/common/blocks/GT_Block_Casings8.java [hunk 1]
```diff
@@ -2,20 +2,27 @@
...
 public class GT_Block_Casings8 extends GT_Block_Casings_Abstract {
+    public static boolean mConnectedMachineTextures = true;
 
     //WATCH OUT FOR TEXTURE ID's
     public GT_Block_Casings8() {
         super(GT_Item_Casings8.class, "gt.blockcasings8", GT_Material_Casings.INSTANCE);
-        for (int i = 0; i < 9; i = (i + 1)) {
-            Textures.BlockIcons.casingTexturePages[1][i+48] = TextureFactory.of(this, i);
+        for (int i = 0; i < 10; i = (i + 1)) {
+            Textures.BlockIcons.casingTexturePages[0][i + 32] = TextureFactory.of(this, i);
```
Comment (GlodBlock) [inline src/main/java/gregtech/common/blocks/GT_Block_Casings8.java:25]: don't change the texture index

**Review by GlodBlock — CHANGES_REQUESTED** [review]

File: src/main/java/gregtech/common/blocks/GT_Block_Casings8.java (hunk 1 above)
Comment (Steelux8) [reply src/main/java/gregtech/common/blocks/GT_Block_Casings8.java:25]: I was unable to fetch the new textures I added if I had them on casingTexturePages[1]. I don't know what I need to change to look in a place other than casingTexturePages[0], which I believe is the default.

**Comment by Steelux8** [comment]
Textures are now fixed. I apologize for my careless mistake.

**Review by GlodBlock — APPROVED** [review]
code wise is okay now. but current gas fuel that fuel value is over 800eu/mb are only nitrobenzene and naquadah gas. naquadah gas is the byproduct for liquid naquadah fuel so it won't be avaible before luv. so only choice is nitrobenzene. i suggest to use 500eu/mb as the threshold so ppl can use ether from ethonal.
also don't forget set the nei catalysts for advanced gas turbine in nei. https://github.com/GTNewHorizons/NotEnoughItems/blob/master/src/main/resources/assets/nei/csv/catalysts.csv

**Comment by Steelux8** [comment]
> code wise is okay now. but current gas fuel that fuel value is over 800eu/mb are only nitrobenzene and naquadah gas. naquadah gas is the byproduct for liquid naquadah fuel so it won't be avaible before luv. so only choice is nitrobenzene. i suggest to use 500eu/mb as the threshold so ppl can use ether from ethonal. also don't forget set the nei catalysts for advanced gas turbine in nei. https://github.com/GTNewHorizons/NotEnoughItems/blob/master/src/main/resources/assets/nei/csv/catalysts.csv

I might tweak recipes to bring more gas fuels above 1M EU, or make new ones altogether. There should be at least another viable fuel in that range, the same way diesels have both CBD and HOG.

**Pull request merged**